users/tracing/groups.go

```diff
@@ -14,14 +14,14 @@ import (
 
 const (
 	assignUser         = "assign_user"
-	saveGroupOp        = "save_group_op"
-	deleteGroupOp      = "delete_group_op"
-	updateGroupOp      = "update_group"
+	saveGroup          = "save_group"
```

Comment:
`save` or `create`?

Reply:
which one you prefer?

Reply:
The consistent one :)

Reply:
I think save should stay as in other places in code
`Create` method is kind of gather info and create object and in the end call `Save`